### PR TITLE
neat: remove dangling `data` property

### PR DESCRIPTION
**Pull request by ianaya89**
- Remove unused `data` property from deploy manifest
- Remove dangling spaces

### CODE REVIEW

**Comment by Slach** [comment]
thanks for contribution, unfortunatelly this PR do nothing

you changed generated file
look to dev folder and try to figure out how exactly it works

Why you struggle with emtpy "data" section?

File: deploy/operator/clickhouse-operator-install-bundle-v1beta1.yaml [hunk 1]
```diff
@@ -3344,7 +3344,6 @@ metadata:
   labels:
     clickhouse.altinity.com/chop: 0.22.2
     app: clickhouse-operator
-data:
```
Comment (Slach) [inline deploy/operator/clickhouse-operator-install-bundle-v1beta1.yaml:3347]: why? which effect?

**Review by Slach — CHANGES_REQUESTED** [review]

File: deploy/operator/clickhouse-operator-install-bundle-v1beta1.yaml (hunk 1 above)
Comment (ianaya89) [reply deploy/operator/clickhouse-operator-install-bundle-v1beta1.yaml:3347]: @Slach thanks for clarifying. Is not an actual issue, but the property looks to be empty (nothing declared inside).
Does not cause problems other than DX (get false warning for linters or that kind of stuff).

**Comment by Slach** [comment]
let's close PR

**Pull request closed without merging**